### PR TITLE
libcouchbase 2.6.0

### PR DESCRIPTION
**Pull request by mnunberg**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

### CODE REVIEW

**Comment by mnunberg** [comment]
Technically speaking, 

```
== Formula/libcouchbase.rb ==
C: 32: 61: Prefer to_s over string interpolation.
```

I can fix it in this PR, however it seems more suitable to have it as a separate commit.

**Comment by mikemcquaid** [comment]
@mnunberg That'd be great, feel free to make a separate commit in this PR 👍

**Comment by mnunberg** [comment]
done!

File: Formula/libcouchbase.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def install
     args << "-DLCB_NO_TESTS=1"
 
     ["libev", "libevent", "libuv"].each do |pname|
-      args << "-DLCB_BUILD_#{pname.upcase}=" + (build.with?("#{pname}") ? "ON" : "OFF")
+      args << "-DLCB_BUILD_#{pname.upcase}=" + (build.with?(pname.to_s) ? "ON" : "OFF")
```
Comment (mikemcquaid) [inline Formula/libcouchbase.rb]: In this case I `pname` is already a string so you can just use `pname`. Also, `package_name` may be a better choice.

Comment (mnunberg) [inline Formula/libcouchbase.rb]: `package_name` or even `pkgname` would occupy more line real estate that `pname`, though. How about `dep`?

Comment (mikemcquaid) [inline Formula/libcouchbase.rb]: 👍

**Comment by dunn** [comment]
Thanks! Merged as f56959a6eb028744496d7565e285f3d42ed6ac2a and 91c2406dc6843df76ad140bd78b10f275f2ca942.

**Pull request closed without merging**